assignment.md

```diff
@@ -12,46 +12,83 @@ If the version control concepts or `git` commands are still confusing to you, co
+* df90802 updated syllabus with presidents day
+* e0029df place holder schedule
+* 7462142 update lecture link for week 3
+* 8fced37 assignment 2 updated
+* 2a2bfd5 version control chapter
+* 13e5722 minor improvements to instructions
+* dd4719c fixed assignment 1 link
+* 0392554 fixed syllabus
+* 48e689c updated syllabus with MLK day
+* e96eba0 added dave's slides link
+* fbb9e8d fixed typo
+* c152f3c added ch2 assignment
+* cd40b26 fixed typo in assignment 1
+* b3455f6 fixed missing link
+* 7cd4587 update links to reflect erlative links
+* fb12bef jekyll config additions for relative links
+* 7ed94e9 dash fix
+* d6f3ec1 fixed link error
+* 2c05357 update links
+* 9586a22 removed type app
+* 00e8ae5 readme and try new size
+* 793782a Set theme jekyll-theme-minimal
+* b7a7152 initial transfer to no annex repo
         ```
 
     - b. Who was the last person to modify `README.md`? (Hint: use `git log` with an argument)
         ```bash
         # git command and output here
+         git log README.md
+                                                                                                                                                                        commit 00e8ae51a08a0f3754ab228f62b51651595c4426                                                                                                                                                                                Author: Xihe Xie <axiezai@gmail.com>                                                                                                                                                                                           Date:   Mon Dec 28 16:01:51 2020 -0500                                                                                                                                                                                                                                                                                                                                                                                                                            readme and try new size   
+                           
         ```
     
     - c. What was the commit message associated with the last modification to the `-Week 2` line of `index.md` file? (Hint: use `git blame` and `git show`, check out their help pages if you are not sure what to do)
+    added ch2 assignment   
 
  5. In the course website repository (or another repository you want to clone from GitHub), modify one of its existing files, do not `git add` or `git commit` yet:
      - a. What happens when you do `git stash`, be specific about what happens to your uncommitted changes after `stash`.?
-       > Answer here
+       > 'Git stash' saves uncommitted changes from the stash log and removes from working directory
      - b. What do you see when you run `git log --all --oneline` after stashing your modification?
-       > Answer here
+       > The stash is shown first before the log.
```

Comment:
indicating there are saved modifications that's not saved as versions or modifications in your git history.